docs/Lua API.rst

```diff
@@ -1287,10 +1287,35 @@ Units module
 
   The unit is of the correct race of the fortress.
 
-* ``dfhack.units.isCitizen(unit)``
+* ``dfhack.units.isCitizen(unit[,ignore_sanity])``
 
   The unit is an alive sane citizen of the fortress; wraps the
-  same checks the game uses to decide game-over by extinction.
+  same checks the game uses to decide game-over by extinction
+  (except for the sanity check).
```

Comment:
```suggestion
  same checks the game uses to decide game-over by extinction.
  You can identify citizens, regardless of their sanity, by passing ``true``
  as the optional second parameter.
```

Reply:
Why drop the information about the sanity check? Shouldn't they know? Especially if the doc makes a point of talking about the closeness to the game's check

Reply:
What did I drop? I thought I just reworded the (except for the sanity check) to make it clearer that you could choose to deviate from the default behavior by passing an optional parameter.

Reply:
It just says it is the same as the game's now. The sanity stuff is an entirely separate statement that can't be inferred as related to the game's default behaviour.

Reply:
I thought the game's default behavior was to not count insane dwarves for the purposes of game over. This flag allows you to modify the default behavior so you can count insane dwarves as citizens

Reply:
According to the function comments, this isn't the case. The game considers the insane to be citizens. It explicitly mentioned it, which is why I did.

Reply:
```
    // Copied from the conditions used to decide game over,
    // except that the game appears to let melancholy/raving
    // dwarves count as citizens.
```

Reply:
though, maybe the berserk/crazed check (part of isSane) is part of the game's check??